docs/cloud/satellites/best-practices.md

```diff
@@ -0,0 +1,90 @@
+# Satellites Best Practices
+
+Earthly Satellites are remote runner instances provided by the Earthly team, offering the ability to perform builds in the cloud while retaining cache between runs. This best practices guide aims to help users optimize their workflow and leverage the full potential of Earthly Satellites.
```

Comment:
reworded the first sentence to align with the text from https://earthly.dev/earthly-satellites
```suggestion
Earthly Satellites are remote build runners hosted by Earthly, which enables building in the cloud with instantly available cache. This best practices guide aims to help users optimize their workflow and leverage the full potential of Earthly Satellites.
```

Maybe we should even link to that page from the docs?

Reply:
Yeah I'll add that too in a follow up commit